packages/contentstack-launch/src/util/common-utility.ts

```diff
@@ -133,4 +133,22 @@ async function selectProject(options: {
   }
 }
 
-export { getOrganizations, selectOrg, selectProject };
+function getLaunchHubUrl(): string {
+  const { cma } = configHandler.get('region') || {};
+  if (!cma) {
+    throw new Error('Region not configured. Please set the region with command $ csdx config:set:region');
+  }
+
+  let launchHubBaseUrl = cma.replace('api', 'launch-api');
+
+  if (launchHubBaseUrl.startsWith('http')) {
+    launchHubBaseUrl = launchHubBaseUrl.split('//')[1];
+  }
+
+  launchHubBaseUrl = launchHubBaseUrl.startsWith('dev11') ? launchHubBaseUrl.replace('dev11', 'dev') : launchHubBaseUrl;
```

Comment:
@aman19K This will work only on dev11 right?

Reply:
@sunil-lakshman , launch is deployed on dev11 only